src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -210,20 +210,44 @@ class MockClientStats {
 
   void SetDelays(std::vector<size_t> times)
   {
-    response_delays.clear();
+    response_delays_.clear();
     for (size_t t : times) {
-      response_delays.push_back(std::chrono::milliseconds{t});
+      response_delays_.push_back(std::chrono::milliseconds{t});
+    }
+  }
+
+  void SetReturnStatuses(std::vector<bool> statuses)
```

Comment:
Can you add a comment describing valid input for this function? e.g. vector size in relation to some other value

Reply:
done